### DISCUSSION TITLE
Searching and pagination

### DISCUSSION THREAD

**Discussion by ghost** [joalla/discogs_client#85] [General] [answered]
I'm trying to search for items, and list just the first item of the first page.
Could somone explain where / how to use the 'per_page' function.

And also, once I receive a paginated object, how do I iterate that paginated item?

**Comment by JOJ0** [comment, marked as answer]
Hi,
you probably have found this chapter in the docs already: https://python3-discogs-client.readthedocs.io/en/latest/quickstart.html#searching

Try to attach an index to the page object to receive the first item only:

```
>>> print(results.page(1)[0])
<Master 22464 "Amon Tobin - Chaos Theory - The Soundtrack To Tom Clancy's Splinter Cell: Chaos Theory">
>>>
```


To iterate, do:
```
for r in results.page(1):
    print(r)
```
